src/test/java/uk/gov/justice/hmpps/prison/api/resource/OpenApiDocsTest.kt

```diff
@@ -91,7 +91,7 @@ class OpenApiDocsTest : ResourceTest() {
       .expectStatus().isOk
       .expectBody()
       .jsonPath("$.components.securitySchemes.bearer-jwt")
-      .isEqualTo(mapOf("type" to "http", "scheme" to "bearer", "bearerFormat" to "JWT", "name" to "Authorization", "in" to "header"))
+      .isEqualTo(mapOf("type" to "http", "scheme" to "bearer", "bearerFormat" to "JWT"))
```

Comment:
not sure why this has changed - think it is something to do with taking spring fox out.  The other projects have only the 3 map items anyway so brings it into line with them

Reply:
actually will be fixed in next PR

Reply:
seems to actually be due to spring docs upgrade from 1.6.1 to 1.6.6